svc-aggregation/system/addcompute.go

```diff
@@ -310,6 +310,72 @@ func (e *ExternalInterface) addCompute(taskID, targetURI, pluginID string, perce
 	if err = PushPluginStartUpData(plugin, pluginStartUpData); err != nil {
 		log.Error(err.Error())
 	}
+	managerURI := "/redfish/v1/Managers/" + plugin.ManagerUUID
+	var managerData map[string]interface{}
+	chassis := make(map[int]map[string]interface{})
+	server := make(map[int]map[string]interface{})
+	managerLinks := make(map[string]interface{})
+	var chassisLink, serverLink, listOfChassis, listOfServer []interface{}
+
+	data, jerr := agmodel.GetResource("Managers", managerURI)
+	if jerr != nil {
+		errorMessage := "error getting manager details: " + jerr.Error()
+		log.Error(errorMessage)
+		return common.GeneralError(http.StatusInternalServerError, response.InternalError, errorMessage,
+			nil, nil), "", nil
+	}
+
+	err = json.Unmarshal([]byte(data), &managerData)
+	if err != nil {
+		errorMessage := "error unmarshalling manager details: " + err.Error()
+		log.Error(errorMessage)
+		return common.GeneralError(http.StatusInternalServerError, response.InternalError, errorMessage,
+			nil, nil), "", nil
+	}
+
+	for index, val := range chassisList {
+		chassis[index] = make(map[string]interface{})
```

Comment:
instead of  this replace it as `listOfChassis=append(listOfChassis,map[string]string{
		"@odata.id":val,
		}
		)`